### PR TITLE
Added Visual Studio Marketplace link.

### PR DESCRIPTION
**Pull request by serranoarevalo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ y: |`md
 `|
 ```
 
+## Installation
+
+The VSCode extension for D2 can be installed using the extension sidebar or by going to the [Visual Studio Marketplace Website](https://marketplace.visualstudio.com/items?itemName=terrastruct.d2) and clicking 'Install'
```
Comment (alixander) [inline README.md:25]: can you change to "Install", and add a period at the end? (after the quotes).

It's also overly verbose in the beginning. Just `You can install using...`

**Review by alixander — CHANGES_REQUESTED** [review]
thank you!

**Review by serranoarevalo — COMMENTED** [review]
Done :)

File: README.md [hunk 2]
```diff
@@ -20,6 +20,10 @@ y: |`md
 `|
 ```
 
+## Install
+
+You can install using the extension sidebar or by going to the [Visual Studio Marketplace Website](https://marketplace.visualstudio.com/items?itemName=terrastruct.d2) and clicking 'Install'.
```
Comment (alixander) [inline README.md:25]: sorry I should've been more clear. I meant replacing `'Install'` with `"Install"`, though I like the installation -> install change too

**Review by alixander — CHANGES_REQUESTED** [review]

**Review by serranoarevalo — COMMENTED** [review]
No problem :)

**Review by alixander — APPROVED** [review]
🙌

**Pull request merged**